src/oop/programs_with_multiple_classes/hard/q3/Book.java

```diff
@@ -0,0 +1,11 @@
+package oop.programs_with_multiple_classes.hard.q3;
+
+public class Book {
+    private int count;
+    private String type;
+
```

Comment:
Can we add a simple method to this class which has them increment the count of books by 1? So if the count is 5, it would become 6?